source/redemption_api.rst

```diff
@@ -25,6 +25,7 @@ channel         No        ``2`` for POS, ``3`` for kiosk, ``4`` for web, ``5`` f
 stamps          No        Integer value indicating the stamps required for a flexible reward
 extra_data      No        JSON object containing any additional data
 qty             No        A number indicating quantity for reward value
+request_id      No        Request ID string, needed if using PIN authorization method
```

Comment:
This field is needed if authorization is PIN authorization is enabled.

Reply:
Hmm ini saya ralat sedikit jadi `This field is needed if PIN authorization is enabled` ya